.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -1,23 +1,34 @@
 ---
 name: Report a bug
 about: Report a bug to help us improve.
-title: "[what is the issue?] in [where is the issue?]"
-labels: "status:triage"
-assignees: ""
+title: [Version]
+labels: Bug
+assignees: ''
+
 ---
 
-**Describe the bug; what happened?**
+**Describe the bug**
+A clear and concise description of what the bug is.
 
-**What are the steps to reproduce the issue?**
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
I think it's a good idea to add how often could this bug be reproduced. Something like
```
....
4. See error
This bug is [always/occasional/seldom] reproducible. 
```